jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -633,28 +641,6 @@ public static SenderConfiguration fromEnv() {
               .build();
     }
 
-    private Interceptor getAddTokenInterceptor(final String authToken) {
-      return getAuthInterceptor("Bearer " + authToken);
-    }
-
-    private Interceptor getAddBasicAuthInterceptor(final String username, final String password) {
-      return getAuthInterceptor(Credentials.basic(username, password));
-    }
-
-    private Interceptor getAuthInterceptor(final String headerValue) {
-      return new Interceptor() {
-        @Override
-        public Response intercept(Chain chain) throws IOException {
-          return chain.proceed(
-                  chain.request()
-                          .newBuilder()
-                          .addHeader("Authorization", headerValue)
-                          .build()
-          );
-        }
-      };
-    }
-
     public static class Builder {
```

Comment:
can we deprecate this class and just use SenderConfiguration directly?

Reply:
I am definitely +1 on that, I will try to do it